### PR TITLE
User Notifications Part 1

### PR DESCRIPTION
**Pull request by josephkane**
* refine user notifications for all interactions relating to logging in and new/edit cloud accounts

* rest of app will be in subsequent prs

### CODE REVIEW

**Comment by taaraora** [comment]
Is it possible unify methods ```displayError``` and ```displaySuccess```?
Also why we do not make first argument of Notifications.display(kind, header, body) to accept constant or enum?

**Comment by josephkane** [comment]
@taaraora `displayError` has to parse an error and get info to display from possibly 2 different keys, where `displaySuccess` just displays a message.
>Also why we do not make first argument of Notifications.display(kind, header, body) to accept constant or enum?
This is a good idea, but we have plans to replace this entire notification system in the future, this is more of a bug fix pr

**Comment by taaraora** [comment]
> @taaraora `displayError` has to parse an error and get info to display from possibly 2 different keys, where `displaySuccess` just displays a message.

Sorry, I wanted to say that there are multiple instances of ```displayError```  methods and May be it is possible to make single `displayError`  function? The same is applicable for all instances of  ```displaySuccess```.

File: cmd/ui/assets/src/app/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -103,6 +104,19 @@ export class DashboardComponent implements OnInit, OnDestroy {
...
   }
 
+  displayError(err) {
+    let msg: string;
+
+    if (err.userMessage) {
+      msg = err.userMessage;
+    } else {
+      msg = err.error;
+      // msg = err.message;
```
Comment (taaraora) [inline cmd/ui/assets/src/app/dashboard/dashboard.component.ts]: unnecessary comment

**Comment by stgleb** [comment]
please fix conflicts @josephkane

**Comment by josephkane** [comment]
@stgleb done

**Pull request merged**